### ISSUE TITLE
play audio on call

### ISSUE THREAD

**Issue by updesh31** [Ai-Thinker-Open/GPRS_C_SDK#178]
## 1. SDK version（SDK 版本）

{

SW_V2112_csdk

}

---

## 2. In what kind of operation problems appear, and how to reproduce the problem ?（什么样的操作步骤问题会出现，是否是稳定复现，如何复现问题？）

{

I stored an audio (amr) file in module successfully.How can I play it at the time of call?

}

---

**Comment by Neutree** [comment]
can't play audio while making a call

**Comment by updesh31** [comment]
At present I am using Mobiletek L206C module in which by AT+CMEDPLAY=1,1.amr,1,100 I can play voice response (not at the time of making call but after answer the call). So I only want to play voice after answering the call.

**Comment by updesh31** [comment]
Any function to play saved voice after answering????

**Comment by Neutree** [comment]
refer to mp3 play, but parameters different, see the header file (api_audio.h）

**Comment by updesh31** [comment]
After start answering I used "AUDIO_Error_t ret = AUDIO_Play(path,AUDIO_TYPE_AMR,PlayCallback);" function which gives error 13 (AUDIO_ERROR_BUSY_CALLING).

**Comment by Neutree** [comment]
yes, like most Feature phone, it can't play music(audio) when making a call

**Comment by updesh31** [comment]
Is it possible in future?
